spec/features/resource_manager_manages_schedules_spec.rb

```diff
@@ -156,14 +156,22 @@ def and_the_guider_has_the_changed_time_slots
     @schedule.reload
 
     first_slot = @schedule.slots.first
-    expect(first_slot.day).to eq 'Wednesday'
-    expect(first_slot.start_at).to eq '11:00'
-    expect(first_slot.end_at).to eq '12:10'
+    expect(first_slot.attributes).to include(
```

Comment:
You could tidy this up a little with the `have_attributes` matcher. At the very least it would allow you to use symbol keys.